.vscode/c_cpp_properties.json

```diff
@@ -4,11 +4,19 @@
             "name": "Win32",
             "includePath": [
                 "C:\\Users\\dilin\\AppData\\Local\\Arduino15\\packages\\AZ3166\\tools\\**",
-                "C:\\Users\\dilin\\AppData\\Local\\Arduino15\\packages\\AZ3166\\hardware\\stm32f4\\1.6.5\\**"
+                "C:\\Users\\dilin\\AppData\\Local\\Arduino15\\packages\\AZ3166\\hardware\\stm32f4\\1.6.5\\**",
```

Comment:
I believe we should not append personal lib path in the repo. Could you help remove this file from the repo.